app/utils/thread/index.ts

```diff
@@ -6,21 +6,20 @@ import {getPreferenceValue} from '@helpers/api/preference';
 
 import type PreferenceModel from '@typings/database/models/servers/preference';
 
-export function processIsCRTEnabled(preferences: PreferenceModel[]|PreferenceType[], config?: ClientConfig): boolean {
+export function processIsCRTEnabled(preferences: PreferenceModel[]|PreferenceType[], configValue?: string, featureFlag?: string): boolean {
```

Comment:
The reasoning to keep it like this is because we know is regarding CRT, but we have two "configurations" affecting: the one in the config, and the one in the feature flag. Therefore the naming.